pkg/api/v1alpha1/cluster.go

```diff
@@ -265,7 +265,10 @@ type kindObject struct {
 // ParseClusterConfigFromContent unmarshalls an API object implementing the KindAccessor interface
 // from a multiobject yaml content. It doesn't set defaults nor validates the object.
 func ParseClusterConfigFromContent(content []byte, clusterConfig KindAccessor) error {
-	r := yamlutil.NewYAMLReader(bufio.NewReader(bytes.NewReader(content)))
+	kubeVersionRegex := regexp.MustCompile(constants.KubernetesVersionRegex)
+	sanitizedContent := kubeVersionRegex.ReplaceAllString(string(content), constants.QuotedKubernetesVersionRegexReplacement)
```

Comment:
Better if we can call a function we defined here rather than the strings function.